packages/executor/src/modules/eth.ts

```diff
@@ -413,7 +413,7 @@ export class Eth {
       const contract = EntryPoint__factory.connect(addr, this.provider);
       try {
         const blockNumber = await this.provider.getBlockNumber();
-        let fromBlockNumber = blockNumber - 15000; // limit query to 15k blocks, otherwise it throw timeout error
+        let fromBlockNumber = blockNumber - this.config.receiptLookupRange;
```

Comment:
Should we retain a default value?

Reply:
@ch4r10t33r yes, the default value is set in the packages/executor/src/config.ts file. It's 1024 blocks by default

https://github.com/etherspot/skandha/pull/58/files#diff-314388631546e13ac73d8334a0e999e4540a222a466f1978b15cfa4e82f47f5bR115